src/pages/index.js

```diff
@@ -158,7 +221,74 @@ function renderCard(item, method = "prepend") {
   cardsList[method](cardElement);
 }
 
-initialCards.forEach(item => renderCard(item, "append"));
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cards]) => {
+    // Store current user data
+    currentUser = userData;
+
+    // Update profile with user data
+    profileName.textContent = userData.name;
+    profileDesc.textContent = userData.about;
+    // Update avatar from server data
+    document.querySelector(".profile__avatar").src = userData.avatar;
```

Comment:
`document.querySelector(".profile__avatar")`

You've already found the avatar in line 66 as `profileAvatar`. Please, reuse it here

```js

profileAvatar.src = userData.avatar;
```